File: Day 3/2 Palindrome String/solution.js
function isPalindrome(str) {
  let i = 0;
  let j = str.length - 1;

  while (i < j) {
      if (str[i] !== str[j]) {
          return false;
      }

      i++;
      j--;
  }

  return true;
}

File: Day 3/2 Palindrome String/solution.cs
public static bool IsPalindrome(string str)
{
    int left = 0;
    int right = str.Length - 1;
    
    while (left < right)
    {
        if (str[left] != str[right])
        {
            return false;
        }
        
        left++;
        right--;
    }
    
    return true;
}
